### PR TITLE
DOCS-7655 Adds Code Analysis Troubleshooting Doc Page

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do? What is the motivation?

Adds a Troubleshooting page to the Code Analysis section.

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/code_analysis/troubleshooting/_index.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  text: "Learn about Software Composition Analysis"
+---
+
+## Overview
+
+If you experience issues setting up or configuring Datadog Code Analysis, use this page to start troubleshooting. If you continue to have trouble, [contact Datadog Support][1].
+
+## Static Analysis
+
+For issues with the Datadog Static Analyzer, include the following information in a bug report to Support as well as your customer success manager.
```
Comment (estherk15) [inline content/en/code_analysis/troubleshooting/_index.md:23]: To verify, I've seen this capitalized
```suggestion
For issues with the Datadog Static Analyzer, include the following information in a bug report to Support as well as your Customer Success Manager.
```

**Review by estherk15 — APPROVED** [review]
Looks good!

File: content/en/code_analysis/troubleshooting/_index.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+## Software Composition Analysis
+
+For issues with Datadog Software Composition Analysis, include the following information in a bug report to Support as well as your customer success manager.
+
+- The output of your SCA tool (such as CLI) that is run locally or in a CI/CD pipeline
+- The SBOM file produced (if there are any available)
+- The URL of your repository (public or private)
+- The name of the branch you ran the analysis on
+- The list of dependency files in your repository (e.g. package-lock.json, requirements.txt, pom.xml)
```
Comment (alai97) [inline content/en/code_analysis/troubleshooting/_index.md:56]:
```suggestion
- The list of dependency files in your repository (such as `package-lock.json`, `requirements.txt`, or `pom.xml`)
```

File: content/en/code_analysis/troubleshooting/_index.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+**Note:** If you are using [Static Analysis and GitHub Actions][2], set the [`enable_performance_statistics`][3] parameter to true.
+
+### Blocking issues
+
+If you are experiencing issues unrelated to performance or if the Datadog Static Analyzer fails to exit, run the Datadog Static Analyzer with the `--debug true --performance-statistics` flag.
+
+## Software Composition Analysis
+
+For issues with Datadog Software Composition Analysis, include the following information in a bug report to Support as well as your customer success manager.
```
Comment (alai97) [inline content/en/code_analysis/troubleshooting/_index.md:50]:
```suggestion
For issues with Datadog Software Composition Analysis, include the following information in a bug report to Support as well as your Customer Success Manager.
```

**Review by markazerdd — APPROVED** [review]

**Pull request merged**